.fixtures.yml

```diff
@@ -10,5 +10,8 @@ fixtures:
       repo: git://github.com/puppetlabs/puppetlabs-yumrepo_core
       ref: 1.0.1
       puppet_version: ">= 6.0.0"
+    archive:
+      repo: git://github.com/voxpupuli/puppet-archive.git
+      ref: 'v3.0.0'
```

Comment:
why the not latest here, too?

Reply:
The pattern we've taken is lowest supported goes in `.fixtures.yml` and latest goes in `.fixtures-latest.yml`.

Reply:
please don't mark conversation started by others as resolved, they may have more questions.

if it's latest, should we leave the ref out and have it actually pull the latest of that module?

Reply:
Only reason I left ref out was we have documented ranges we support so the range we support had upper bound in `.fixtures-latest.yml` which helps avoid future updates to modules breaking our tests.  I think we just need to document we only support puppet/archive of essentially 3.x. What do you think?

Reply:
We can document the version requirements in metadata.json. Think that the .fixtures-latest should not have ref's though should also be allowed to fail. This will inform us when say archive goes to 4.x and we need to update the metadata to show it also works or it fails and we choose to do work or not to support it.

Reply:
I like idea of removing ref but should we just document `archive` requirement in README and not metadata.json since it's only needed for Windows?  We did something similar for `apt` module since only needed for Debian based systems.  We documented that one as a soft dependency for apt systems.

Reply:
I removed all refs from `.fixtuers-latest.yml` and added version range for `puppet-archive` module to README, let me know if really want this in metadata.json.

Reply:
Ah yes, just document archive is needed with a link and a version greater/less than just like metadata.json would